system-test/testTypes.ts

```diff
@@ -46,3 +47,24 @@ export interface Test {
   row_keys_read: {};
   createReadStream_options: GetRowsOptions;
 }
+
+interface CreateReadStreamResponse {
+  row_keys?: string[];
+  end_with_error?: number;
+  error_message?: string;
+}
+
+export interface CreateReadStreamRequest {
+  rowKeys: string[];
+  rowRanges: google.bigtable.v2.IRowRange[];
+  rowsLimit?: number;
+}
+export interface ReadRowsTest {
+  createReadStream_options?: GetRowsOptions;
+  max_retries: number;
+  responses: CreateReadStreamResponse[];
+  request_options: CreateReadStreamRequest[];
+  error: number;
+  row_keys_read: string[][];
+  name: string;
+}
```

Comment:
There is a weird mix of camel case and snake case here because they match the keys already contained in the json file.